components/Molecule/main-banner.html

```diff
@@ -20,35 +19,33 @@
             display: flex;
             align-items: center;
             justify-content: space-around;
-            background: linear-gradient(to right, #cc5333, #23074d);
             width: 100%;
 
         }
 
-        ul {
-            width: 80%;
+        .main-banner-nav-list{
             margin: 0 auto;
             padding: 10px;
         }
 
-        ul li {
+        .main-banner-nav-list li {
             list-style: none;
             display: inline-block;
             padding: 20px;
             font-size: 20px;
-            font-family: "Times New Roman", Times, serif;
+            font-family: 'Lexend Deca'
```

Comment:
@ayabaknu can you add `sans-serif` as a fallback font? That way if Lexend Deca is not available, the font will go to sans serif. 

`font-family: 'Lexend Deca', sans-serif`